src/s3/objectCreated.ts

```diff
@@ -0,0 +1,41 @@
+import { S3CreateEvent, S3Handler } from 'aws-lambda';
+import dependencies, { Container } from '../dependencies';
+
+const isMetadataFile = (key: string) => key.endsWith('.json');
+const getDocumentIdFromKey = (key: string) => key.split('/')[0];
+
+const createHandler: (container: Container) => S3Handler = (container: Container) => {
+  const {
+    logger,
+    indexDocument: indexer,
+  } = container;
```

Comment:
Might refactor this to only take the deps it needs rather than taking the whole dependencies container and then pulling out bits.